homeassistant/components/konnected/panel.py

```diff
@@ -64,7 +65,9 @@ def __init__(self, hass, config_entry):
         self.hass = hass
         self.config_entry = config_entry
         self.config = config_entry.data
-        self.options = config_entry.options
+        self.options = config_entry.options or config_entry.data.get(
```

Comment:
If a user had options specified in configuration.yaml we will utilize them automatically until an updated set of options is created via the options flow.